README.md

```diff
@@ -447,7 +447,7 @@ The full API spec is available [here](https://pkg.go.dev/github.com/textileio/go
 import "github.com/textileio/go-threads/net/api/client"
 ...
 
-net, err := client.NewClient("/ip4/127.0.0.1/tcp/6006", grpc.WithInsecure())
```

Comment:
Good catch. As the client communicates with regular gRPC server, TCP addr should be passed in instead of multiaddr. Correct me if I'm wrong @sanderpick

Reply:
yep, correct! thanks @bbigras